submissions/opeyemiPeter/html/index.html

```diff
@@ -1,141 +1,112 @@
 <!DOCTYPE html>
```

Comment:
Changes were not asked to be made to the product page. This would come in in another task. Yu should revert your changes and remove the images added @opeyemipeter1759